### PR TITLE
More nwfind fixes

### PR DESCRIPTION
**Pull request by ddutt**
## Description

Network find had a bunch of issues associated with empty output if s VLAN was specified, or sometimes when start/end times were specified and so on. This PR addresses all those issues.

## Type of change

- Bug fix (non-breaking change which fixes an issue)

## Double Check

- [X] I have read the comments and followed the [CONTRIBUTING.md](https://github.com/netenglabs/suzieq/blob/master/CONTRIBUTING.md).
- [X] I have explained my PR according to the information in the comments or in a linked issue.
- [X] My PR source branch is created from the `develop` branch.
- [X] My PR targets the `develop` branch.
- [X] All my commits have `--signoff` applied

### CODE REVIEW

File: suzieq/engines/pandas/network.py [hunk 1]
```diff
@@ -120,7 +121,7 @@ def _find_bond_members(self, namespace: str, hostname: str,
         # get list  of namespaces, hostnames and ifnames to get info for
 
         ifdf = self._get_table_sqobj('interfaces',
```
Comment (claudiolor) [inline suzieq/engines/pandas/network.py:123]: I think it is important to comment the reason why we we set these values of start time and end time.

File: suzieq/engines/pandas/network.py [hunk 2]
```diff
@@ -157,39 +158,31 @@ def _find_addr_arp(self, addr: str, **kwargs) -> pd.DataFrame:
...
-                    'ipAddress': row.ipAddress,
-                    'vlan': '-',
-                    'macaddr': row.macaddr,
-                    'ifname': row.oif,
-                    'type': 'bridged',
-                    'l2miss': False,
-                    'timestamp': row.timestamp
-                })
-                continue
+            active = getattr(row, 'active', False)
```
Comment (claudiolor) [inline suzieq/engines/pandas/network.py:161]: If possibile, it would be nice renaming `row` to something like `arp_row`

**Review by claudiolor — APPROVED** [review]

File: suzieq/cli/sqcmds/command.py [hunk 3]
```diff
@@ -316,10 +316,10 @@ def _gen_output(self, df: pd.DataFrame, json_orient: str = "records",
...
             if 'active' in df.columns:
                 df['active'] = np.where(df.active, '+', '-')
-            if (not (self.start_time or self.end_time) and
-                'timestamp' not in self.columns and not dont_strip_cols and
-                    'timestamp' in df.columns and 'timestamp' in cols):
-                cols.remove('timestamp')
+            if (not (self.view == "all" or self.columns == ['*'] or
+                     (self.start_time and self.end_time) or dont_strip_cols)):
+                if 'timestamp' in cols:
+                    cols.remove('timestamp')
```
Comment (LucaNicosia) [inline suzieq/cli/sqcmds/command.py:322]:
```suggestion
            if ((not (self.view == "all" or self.columns == ['*'] or
                     (self.start_time and self.end_time) or dont_strip_cols)) and 
                 'timestamp' in cols):
                cols.remove('timestamp')
```

File: suzieq/engines/pandas/network.py [hunk 4]
```diff
@@ -157,39 +158,31 @@ def _find_addr_arp(self, addr: str, **kwargs) -> pd.DataFrame:
...
 
                     tmpres.update({
+                        'active': active,
                         'namespace': row.namespace,
                         'hostname': row.hostname,
-                        'vrf': ifdf.master.unique().tolist()[0] or 'default',
+                        'vrf': row_vrf,
                         'ipAddress': row.ipAddress,
-                        'vlan': ifdf.vlan.astype(str).unique().tolist()[0],
+                        'vlan': ifdf.vlan.unique().tolist()[0],
```
Comment (LucaNicosia) [inline suzieq/engines/pandas/network.py:185]: Store the list of unique vlans in variable. This value is used in multiple places

**Review by claudious96 — APPROVED** [review]

**Pull request merged**